scripts/run_mypy_checks.py

```diff
@@ -177,7 +198,8 @@ def main(args: Optional[List[str]] = None) -> int:
         # https://stackoverflow.com/q/10095037 for more details.
         sys.path.insert(1, directory)
 
-    install_third_party_libraries(parsed_args.skip_install)
+    if not feconf.OPPIA_IS_DOCKERIZED:
```

Comment:
I am here trying to maintain the behaviour of the script what it is performing, but yes, this should be removed and the tests can be simply executed with the direct command from the makefile, maybe later after in the future enhancements

Reply:
We already have the `--skip-install` flag in the script, can't we use it directly?

Reply:
no, because at many places the file is just simply imported and then its main function is executed without any flag. refer [here](https://github.com/oppia/oppia/blob/develop/scripts/run_backend_tests.py#L70), and many other places too

Reply:
@DubeySandeep, we will be removing the `OPPIA_IS_DOCKERIZED` flag in an upcoming PR (which will be opened by the 20th). This is just a temporary variable to make sure that the Docker and Python setup work.

Reply:
removed the flag in import statement